proto/Cargo.toml

```diff
@@ -18,11 +18,11 @@ json = ["base64", "serde"]
 
 [dependencies]
 base64 = { version = "^0.22", optional = true }
-prost = "^0.13"
+prost = "^0.14"
 serde = { version = "^1.0.203", features = ["derive"], optional = true }
 
 [build-dependencies]
-prost-build = "^0.13"
+prost-build = "^0.14"
```

Comment:
Looks like the auto-generated code under `proto/src/codegen` needs to be regenerated after the upgrade.  (IIRC you can trigger that by building with `PROTOC` pointing to a valid `protoc`.)